controllers/utils.js

```diff
@@ -33,10 +33,11 @@ function hasUserSignedUpForRelay(user) {
   return false;
 }
 
-function getExperimentBranch(req, language, sorterNum) {
+function getExperimentBranch(req, sorterNum = false, language = false) {
```

Comment:
This also reverses the order of the parameters? I don't see the calls to the function changed to match?

Reply:
It was commented out/broken previously. I had added back in the language logic for future use. It's now working. For the current experiment, there was no language filtering, so no args is okay.